peps/pep-0728.rst

```diff
@@ -141,424 +142,422 @@ in TypeScript achieve this:
 
 This proposal aims to support a similar feature without introducing general
 intersection of types or syntax changes, offering a natural extension to the
-existing type consistency rules.
-
-We propose that we add an argument ``closed`` to TypedDict. Similar to
-``total``, only a literal ``True`` or ``False`` value is allowed. When
-``closed=True`` is used in the TypedDict type definition, we give the dunder
-attribute ``__extra_items__`` a special meaning: extra items are allowed, and
-their types should be compatible with the value type of ``__extra_items__``.
+existing assignability rules.
 
-If ``closed=True`` is set, but there is no ``__extra_items__`` key, the
-TypedDict is treated as if it contained an item ``__extra_items__: Never``.
+We propose that we add a class parameter ``extra_items`` to TypedDict.
+It accepts a :term:`type expression` as the argument; when it is present,
```

Comment:
Does this link work? Feel you need to put `typing:` in there somewhere to intersphinx to the typing spec.

Reply:
The link did work. I updated those references anyway because namespacing these with `typing:` seems to be helpful.